doc/kubernetes/modules/ROOT/pages/automation-using-tool-task.adoc

```diff
@@ -0,0 +1,97 @@
+= Automation using the tool _task_
+:description: Few details around the Automation related to provisioning and configuration of the Kubernetes deployment.
+
```

Comment:
I wonder it this file should be moved to the folder `util` and should be named `task.adoc`. It would then also be moved in the `nav.adoc` to to that menu section. Please let me know what you think about this idea.

Reply:
Yup, I agree to this change. Thanks for making the commit around this change.